contracts/libs/Actions.sol

```diff
@@ -275,6 +281,13 @@ library Actions {
         require(_args.actionType == ActionType.Call, "Actions: can only parse arguments for call actions");
         require(_args.sender != address(0), "Actions: target address cannot be address(0)");
 
-        return CallArgs({owner: _args.owner, callee: _args.sender, data: _args.data});
+        return
+            CallArgs({
+                owner: _args.owner,
+                callee: _args.sender,
+                vaultId: _args.vaultId,
+                msgValue: _args.amount,
```

Comment:
I think we should call it value instead of `msgValue`. message value implies that this is the same as msg.value, but it doesn't. If a user pass in 2 call actions each with value = 5, the msg.value will need to be 10. This may get quite confusing.